packages/react-vapor/src/components/tab/Tab.tsx

```diff
@@ -72,14 +76,21 @@ export const Tab: React.FunctionComponent<ITabProps> = ({
                 className={classNames('tab', {
                     enabled: !disabled,
                     disabled: disabled,
-                    active: isActive,
+                    'body-m': isActive,
+                    'body-m-book': !isActive,
                 })}
                 onClick={handleSelect}
                 disabled={disabled}
             >
-                {children}
+                {icon ? <Svg svgName={icon} svgClass={'tab-icon mod-16'} /> : null}

```

Comment:
Have you tried displaying a stroked icon? I think perhaps it will require the class `mod-stroke` here. For example using icon `crawlingBotStroked16`

Reply:
i'll try it :+1: